Docs for Decoupled TaintManager from NodeLifeCycleController (KEP-3902) (#43555)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates.md

| `SchedulerQueueingHints` | `false` | Alpha | 1.28 | |
| `SecurityContextDeny` | `false` | Alpha | 1.27 | |
| `SidecarContainers` | `false` | Alpha | 1.28 | |

> | `SeparateTaintEvictionController` | `true` | Beta | 1.29 | |

Reviewer: Move this entry above the previous one for ordering's sake?

| `ServiceAccountTokenPodNodeInfo` | `false` | Alpha | 1.29 | |
| `SidecarContainers` | `false` | Alpha | 1.28 | 1.28 |
| `SidecarContainers` | `true` | Beta | 1.29 | |

> | `SeparateTaintEvictionController` | `true` | Beta | 1.29 | |

Reviewer: This entry should not appear here. Please make sure the rows are still in alphabetic order.

  for all workloads.
  The seccomp profile is specified in the `securityContext` of a Pod and/or a Container.
- `SecurityContextDeny`: This gate signals that the `SecurityContextDeny` admission controller is deprecated.

> - `SeparateTaintEvictionController`: Enables running `TaintEvictionController` - that performs [Taint Based Evictions](/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions) - in a separated controller from `NodeLifecycleController`.

Reviewer: Please wrap the long lines manually.
Maybe we can add a sentence or two articulating the user perceivable impacts from this.
As a user I need to know whether/when to turn this gate off.
```suggestion
- `SeparateTaintEvictionController`: Enables running `TaintEvictionController`
  that performs [Taint Based Evictions](/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions)
  in a separated controller from `NodeLifecycleController`.
```

  for all workloads.
  The seccomp profile is specified in the `securityContext` of a Pod and/or a Container.
- `SecurityContextDeny`: This gate signals that the `SecurityContextDeny` admission controller is deprecated.
- `SeparateTaintEvictionController`: Enables running `TaintEvictionController`,

>   that performs [Taint Based Evictions](/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions),

Reviewer 2: ```suggestion
  that performs [Taint-based Evictions](/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions),
```

>   in a separated controller from `NodeLifecycleController`. When this feature is

Reviewer: ```suggestion
  in a controller separated from `NodeLifecycleController`. When this feature is
```

>   enabled, users can optionally disable Taint Based Eviction setting the

Reviewer 2: ```suggestion
  enabled, users can optionally disable Taint-based Eviction by setting the
```

>   `--controllers=-taint-eviction-controller` flag in Kube Controller Manager.

Reviewer: ```suggestion
  `--controllers=-taint-eviction-controller` flag on the `kube-controller-manager`.
```
